both/collections/round.js

```diff
@@ -30,6 +30,13 @@ _.extend(Round.prototype, {
     assert(this.eventCode);
     return wca.eventByCode[this.eventCode].solveTimeFields;
   },
+  roundCode: function() {
+    if(this.isLast()) {
+      return this.softCutoff ? 'c' : 'f';
+    } else {
+      return (this.softCutoff ? 'degc' : '123f')[this.nthRound-1];
```

Comment:
Can we move this logic into [wca.js](https://github.com/cubing/ccm/blob/master/both/lib/wca.js)?